pkg/kubelet/dockertools/docker_manager_test.go

```diff
@@ -2062,8 +2062,8 @@ func TestCheckVersionCompatibility(t *testing.T) {
 }
 
 func TestVersion(t *testing.T) {
-	expectedVersion := "1.8.1"
-	expectedAPIVersion := "1.20"
+	expectedVersion := "1.10.3"
+	expectedAPIVersion := "1.10.4-28.gitf476348.fc23"
```

Comment:
`1.10.4-28.gitf476348.fc23` is not api version. I think this function is just testing whether `Version()` and `APIVersion()` return version correctly.

You can test this in `TestCheckVersionCompatibility`. :)